### DISCUSSION TITLE
Embedded PDF

### DISCUSSION THREAD

**Discussion by 3813127458** [sumatrapdfreader/sumatrapdf#4276] [Ideas]
A missing feature of the current release, in contrast to old version 3.1.2 is: 
Open embedded PDF in new tab or new window by simple clicks on the embedded PDF symbol.

**Comment by kjk** [comment]
Have you tried with latest pre-release https://www.sumatrapdfreader.org/prerelease ?

If it happens please open a bug and provide a test file via https://github.com/sumatrapdfreader/sumatrapdf/issues/new?assignees=&labels=&projects=&template=bug_report.md&title=

**Comment by 3813127458** [comment]
**SumatraPDF version**
- Version 3.6.16272, pre-release

**Describe the bug**
Maybe it's not a bug but missing feature:
The embedded PDF should open by double click on the PDF symbol and optional "Save As" using right mouse button context menu. Other file types, such as IGES in the example above, should be possible "Save As" using right mouse button context menu, in case the operating system does not offer a designated application.

**To Reproduce**
Steps to reproduce the behavior:
[main.pdf](https://github.com/sumatrapdfreader/sumatrapdf/files/15447091/main.pdf)

1. load main.pdf
2. double-click on a PDF Acrobat symbol
4. result: the embedded PDF does not open

**Reply by kjk** [reply]
It works in pre-release now: click embedded file to open in a tab

**Comment by 3813127458** [comment]
Works great! Thanks! ... Now this really convenient feature is back.
